bin/reflash

```diff
@@ -135,10 +135,14 @@ backup_usb_key()
             set -o xtrace
         fi
 
-        /opt/smartdc/bin/sdc-usbkey mount >/dev/null || {
-            echo "unable to mount USB key" >&2
-            exit 1
-        }
+        if [[ -z \$(mount | grep "^\/mnt\/usbkey ") ]]; then
```

Comment:
Can we add a comment as to why we're double-checking here (since sdc-usbkey checks itself too) ?

Reply:
Yes.